contracts/ronin/validator/RoninValidatorSet.sol

```diff
@@ -8,7 +8,7 @@ import "./CoinbaseExecution.sol";
 import "./SlashingExecution.sol";
 
 contract RoninValidatorSet is Initializable, CoinbaseExecution, SlashingExecution {
-  constructor() {
+  constructor() payable {
```

Comment:
This contract is behind a proxy? Do we need to allow it receive ether on construction?

Reply:
declare constructor as payable will cut off opcodes for requiring msg.value != 0
reference: https://github.com/OpenZeppelin/openzeppelin-contracts/issues/3059

However, I see no changes in the code size after compiling so I will remove it

Reply:
resolved.